app/src/main/java/com/quran/labs/androidquran/presenter/data/QuranDataPresenter.kt

```diff
@@ -98,20 +95,6 @@ class QuranDataPresenter @Inject internal constructor(
     }
   }
 
-  private fun copyLocalDataIfNecessary(status: QuranDataStatus): Single<QuranDataStatus> {
```

Comment:
was not used today - but was planning on using for warsh, so moved this as a warsh upgrade and removed from here.